verticaltabs.jsm

```diff
@@ -101,19 +101,25 @@ function VerticalTabs(window, {newPayload, addPingStats, AppConstants, setDefaul
 VerticalTabs.prototype = {
 
   init: function () {
-    this.BrowserOpenTab = this.window.BrowserOpenTab;
-    this.window.BrowserOpenTab = function () {
-      this.pushToTop = true;
-      this.window.openUILinkIn(this.window.BROWSER_NEW_TAB_URL, 'tab');
-      this.pushToTop = false;
```

Comment:
oh, good. yup you're right. nope, don't need the `BrowserOpenTab` function. Maybe you left in the override code?